src/renderer/i18n/index.js

```diff
@@ -31,20 +32,17 @@ export async function loadLocale(locale) {
       // eslint-disable-next-line n/no-path-concat
       const compressed = await readFile(`${__dirname}/static/locales/${locale}.json.br`)
       const decompressed = await brotliDecompressAsync(compressed)
-      const data = JSON.parse(decompressed.toString())
-      i18n.setLocaleMessage(locale, data)
+      data = JSON.parse(decompressed.toString())
     } catch (err) {
       console.error(locale, err)
     }
   } else {
     const url = createWebURL(`/static/locales/${locale}.json`)
 
     const response = await fetch(url)
-    const data = await response.json()
-    i18n.setLocaleMessage(locale, data)
+    data = await response.json()
   }
+  i18n.global.setLocaleMessage(locale, data)
```

Comment:
Could you please revert this changes or add an early return, as the current code will attempt to set the local messages, even if an error occurred while reading them from the file system, which the previous code didn't.